### ISSUE TITLE
Add subcommand to list the path to an installation by name

### ISSUE THREAD

**Issue by sanmiguel** [kerl/kerl#185]
Would be super helpful right now in adding Travis support to be able to ask kerl for the location of an install by name, rather than having to make assumptions about paths etc.
For example:

```
$ kerl path 19.2.3
/home/user/kerl/19.2.3
```

Alternatively it could be an optional arg to `kerl list install[ations]`:

```
$ kerl list installations 19.2.3
/home/user/kerl/19.2.3
```

But this makes the output inconsistent because `list installations` gives the name and path, not just the path...

Thoughts?

**Comment by mrallen1** [comment]
👍 

I really like this idea. 

Should be just a quick wrapper around the `get_active_path()` function which already exists.

**Comment by sanmiguel** [comment]
Ah just re-read this and realised it's not quite what I meant. We can already get the active path from the env, like you said. What I'm after is a way to get the path to an installation from it's name, regardless of which installation (if any) is currently active.

**Comment by sanmiguel** [comment]
- [ ] add `path` subcommand
- [ ] add `path <install>` subcommand
- [ ] extend zsh_completion
- [ ] extend bash completion

**Comment by sanmiguel** [comment]
This'll get fixed by #183 now that I've rolled the PR formerly known as #190 into it.

**Comment by mrallen1** [comment]
@sanmiguel Shouldn't this get closed now that #190 is merged? Or do you want to wait until it's tagged?

**Comment by sanmiguel** [comment]
No you're right, this should get closed now.
